doc/manuals/deprecated.md

```diff
@@ -15,6 +15,9 @@ not mantained or evolved any longer. In particular:
 
 A list of deprecated features and the version in which they were deprecated follows:
 
+* `location` metadata to specify entity location is deprecated in Orion 1.1.0. The alternative way
```

Comment:
As the 'first way' has been deprecated, talking about an 'alternative way' is a little bit funny.
I would choose a stronger word.  Recommended ... no, not very good ...

Perhaps:
The new way (from Orion XXX) of specifying entity location is ...